mmdet/models/dense_heads/corner_head.py

```diff
@@ -0,0 +1,714 @@
+        self.loss_emb = build_loss(loss_emb) if loss_emb is not None else None
+        self.loss_off = build_loss(loss_off) if loss_off is not None else None
+        self.train_cfg = train_cfg
+        self.test_cfg = test_cfg
+
+        self._init_layers()
+
+    def _make_layers(self, out_channels, in_channels=256, feat_channels=256):
+        if out_channels == 0:
+            return nn.Sequential()
+        else:
+            return nn.Sequential(
+                ConvModule(in_channels, feat_channels, 3, padding=1),
+                nn.Conv2d(feat_channels, out_channels, (1, 1)))
+
+    def _init_corner_kpt_layers(self):
+        self.tl_pool, self.br_pool = nn.ModuleList(), nn.ModuleList()
+        self.tl_heat, self.br_heat = nn.ModuleList(), nn.ModuleList()
+        self.tl_off, self.br_off = nn.ModuleList(), nn.ModuleList()
+
+        for _ in range(self.feat_num_levels):
+            self.tl_pool.append(BCPool(self.in_channels, 'top', 'left'))
+            self.br_pool.append(BCPool(self.in_channels, 'bottom', 'right'))
+
+            self.tl_heat.append(
+                self._make_layers(
+                    out_channels=self.num_classes,
+                    in_channels=self.in_channels))
+            self.br_heat.append(
+                self._make_layers(
+                    out_channels=self.num_classes,
+                    in_channels=self.in_channels))
+
+            self.tl_off.append(
+                self._make_layers(
+                    out_channels=self.corner_offset_channels,
+                    in_channels=self.in_channels))
+            self.br_off.append(
+                self._make_layers(
+                    out_channels=self.corner_offset_channels,
+                    in_channels=self.in_channels))
+
+    def _init_corner_emb_layers(self):
+        self.tl_emb, self.br_emb = nn.ModuleList(), nn.ModuleList()
+
+        for _ in range(self.feat_num_levels):
+            self.tl_emb.append(
+                self._make_layers(
+                    out_channels=self.corner_emb_channels,
+                    in_channels=self.in_channels))
+            self.br_emb.append(
+                self._make_layers(
+                    out_channels=self.corner_emb_channels,
+                    in_channels=self.in_channels))
+
+    def _init_layers(self):
+        self._init_corner_kpt_layers()
+        if self.with_corner_emb:
+            self._init_corner_emb_layers()
+
+    def init_weights(self):
+        """
+        -2.19 is the magic number copy from `official github repo
+        <https://github.com/princeton-vl/CornerNet/blob/master/models/py_utils/kp.py#L137>`_ .  # noqa: E501
+        More details can be found in the `issue link
+        <https://github.com/princeton-vl/CornerNet/issues/13>`_ .
+        """
+        for i in range(self.feat_num_levels):
+            self.tl_heat[i][-1].bias.data.fill_(-2.19)
+            self.br_heat[i][-1].bias.data.fill_(-2.19)
```

Comment:
`-2.19` is not a magical number. Use [this](https://github.com/open-mmlab/mmcv/blob/a7bf770168f36d4a4864a666114fb9b93f7e318f/mmcv/cnn/utils/weight_init.py#L63) instead.